src/routes/jira/events/jira-events-uninstall-post.test.ts

```diff
@@ -34,12 +42,29 @@ describe("Webhook: /events/uninstalled", () => {
 				.mockResolvedValue(subscriptions)
 		};
 
+		frontendApp = express();
```

Comment:
Since we are calling the `await JiraEventsUninstallPost(req, res);` directly, don't think we need to setup `frontendApp` anyway. Or if we do, use `supertest(frontendApp)` instead.